examples/curated-paranet-demo.js

```diff
@@ -0,0 +1,260 @@
+const PRIVATE_KEY3 = '0x7c852118294e51e653712a81e05800f419141751be58f605c371e15141b007a6';
+
+const PUBLIC_KEY4 = '0xe5beaB7853A22f054Ef287EA62aCe7A32528b3eE';
+const PRIVATE_KEY4 = '0x9904da7fe786e5d1f8629b565b688425d78053d4325e746c5ad8ac4328248037';
+
+const PUBLIC_KEY5 = '0x8A4673B00B04b59CaC44926ABeDa85ed181fA436';
+const PRIVATE_KEY5 = '0xfb07091daf99c1d493820ae8dcbc439b48b13ca844684bb1dcae27c9e680e62b';
+
+const dkgClient2 = new DKG({
+    environment: ENVIRONMENT,
+    endpoint: OT_NODE_HOSTNAME,
+    port: OT_NODE_PORT,
+    blockchain: {
+        name: 'hardhat2',
+        publicKey: PUBLIC_KEY2,
+        privateKey: PRIVATE_KEY2,
+    },
+    maxNumberOfRetries: 30,
+    frequency: 2,
+    contentType: 'all',
+});
+
+const dkgClient3 = new DKG({
+    environment: ENVIRONMENT,
+    endpoint: OT_NODE_HOSTNAME,
+    port: OT_NODE_PORT,
+    blockchain: {
+        name: 'hardhat2',
+        publicKey: PUBLIC_KEY3,
+        privateKey: PRIVATE_KEY3,
+    },
+    maxNumberOfRetries: 30,
+    frequency: 2,
+    contentType: 'all',
+});
+
+const dkgClient4 = new DKG({
+    environment: ENVIRONMENT,
+    endpoint: OT_NODE_HOSTNAME,
+    port: OT_NODE_PORT,
+    blockchain: {
+        name: 'hardhat2',
+        publicKey: PUBLIC_KEY4,
+        privateKey: PRIVATE_KEY4,
+    },
+    maxNumberOfRetries: 30,
+    frequency: 2,
+    contentType: 'all',
+});
+
+const dkgClient5 = new DKG({
+    environment: ENVIRONMENT,
+    endpoint: OT_NODE_HOSTNAME,
+    port: OT_NODE_PORT,
+    blockchain: {
+        name: 'hardhat2',
+        publicKey: PUBLIC_KEY5,
+        privateKey: PRIVATE_KEY5,
+    },
+    maxNumberOfRetries: 30,
+    frequency: 2,
+    contentType: 'all',
+});
+
+function divider() {
+    console.log('==================================================');
+    console.log('==================================================');
+    console.log('==================================================');
+}
+
+(async () => {
+
+    divider();
+
+    const nodeInfo = await DkgClient.node.info();
+    console.log('======================== NODE INFO RECEIVED');
+    console.log(nodeInfo);
+
+    divider();
+
+    let content = {
+        public: {
+            '@context': ['https://schema.org'],
+            '@id': 'uuid:1',
+            company: 'ParanetOperatorsDAO',
+            user: {
+                '@id': 'uuid:user:1',
+            },
+            city: {
+                '@id': 'uuid:Belgrade',
+            },
+        }
+    };
+
+    divider();
+
+    const paranetAssetResult = await DkgClient.asset.create(content, { epochsNum: 2 });
+    console.log('======================== PARANET KNOWLEDGE ASSET CREATED');
+    console.log(paranetAssetResult);
+
+    divider();
+
+    const paranetOptions = {
+        paranetName: 'FirstParanet',
+        paranetDescription: 'First ever paranet on DKG!',
+        tracToNeuroEmissionMultiplier: 5,
+        incentivizationProposalVotersRewardPercentage: 12.00,
```

Comment:
Not that important, but 10_000 is 100% for these parameters. Would be good to add comments on that perhaps

Reply:
Removed 
``` 
tracToNeuroEmissionMultiplier: 5,
incentivizationProposalVotersRewardPercentage: 12.00,
operatorRewardPercentage: 10.00,
```
because we are not deploying an Incentives contract in curated paranet demo so they do not serve any function.